stdlib/public/SDK/CoreGraphics/CMakeLists.txt

```diff
@@ -8,6 +8,6 @@ add_swift_library(swiftCoreGraphics ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_
   SWIFT_COMPILE_FLAGS "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}"
   LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
   SWIFT_MODULE_DEPENDS ObjectiveC Dispatch Darwin
-  SWIFT_MODULE_DEPENDS_OSX IOKit XPC
+  SWIFT_MODULE_DEPENDS_OSX Dispatch IOKit ObjectiveC os
```

Comment:
This one should probably have _IOS, _TVOS, _WATCHOS entries too. Maybe the script should complain if there's no entry for a particular SDK.